service-request-infno-spec-1.0.0-swagger.json

```diff
@@ -0,0 +1,213 @@
+                  "items" : {
+                    "$ref" : "#/components/schemas/AreaSchema"
+                  }
+                }
+              }
+            }
+          },
+          "400" : {
+            "description" : "bad input parameter"
+          },
+          "404" : {
+            "description" : "Page Not Found"
+          },
+          "500" : {
+            "description" : "Internal Server Error"
+          },
+          "502" : {
+            "description" : "Bad Gateway Error"
+          },
+          "503" : {
+            "description" : "Service Unavailable Error"
+          }
+        }
+      }
+    },
+    "/service-request/image_placeholder/" : {
+      "get" : {
+        "tags" : [ "media" ],
+        "parameters" : [ {
+          "in" : "query",
+          "name" : "labels",
+          "required" : false,
+          "schema" : {
+            "type" : "string"
+          }
+        } ],
+        "responses" : {
+          "200" : {
+            "description" : "OK",
+            "content" : {
+              "application/json" : {
+                "schema" : {
+                  "type" : "array",
+                  "items" : {
+                    "$ref" : "#/components/schemas/ImageRecognitionSchema"
+                  }
+                }
+              }
+            }
+          },
+          "500" : {
+            "description" : "Internal Server Error"
+          }
+        }
+      }
+    }
+  },
+  "components" : {
+    "schemas" : {
+      "ServiceRequestSchema" : {
+        "type" : "object",
+        "required" : [ "id", "sr_status", "type", "suburb" ],
+        "properties" : {
+          "id" : {
+            "type" : "string",
+            "format" : "uuid",
+            "example" : "d290f1ee-6c54-4b01-90e6-d701748f0851"
+          },
+          "sr_status" : {
+            "type" : "string",
+            "example" : 987654567
+          },
+          "type" : {
+            "type" : "string"
+          },
+          "suburb" : {
+            "type" : "string"
+          }
+        }
+      },
+      "AreaSchema" : {
+        "required" : [ "area_stats" ],
+        "properties" : {
+          "name" : {
+            "type" : "string"
+          }
+        }
+      },
+      "StatsSchema" : {
+        "required" : [ "count", "open", "duration_p50", "heatmap_link", "last_updated" ],
+        "properties" : {
+          "count" : {
+            "type" : "integer"
+          },
+          "open" : {
+            "type" : "integer"
+          },
+          "duration_p50" : {
+            "type" : "integer"
+          },
+          "heatmap_link" : {
+            "type" : "string"
+          },
+          "last_updated" : {
+            "type" : "string",
+            "format" : "date-time"
+          }
+        }
+      },
+      "ImageRecognitionSchema" : {
+        "required" : [ "url" ],
+        "properties" : {
+          "url" : {
+            "type" : "array"
+          }
+        }
```

Comment:
I think our response here will be an array of objects of the form:
```json
[
  {
  "type": "string",
  "confidence": "float",
  }
]
```

Reply:
Response has been changed